brd-ios/breadwallet/src/ViewControllers/LoginViewController.swift

```diff
@@ -445,6 +445,7 @@ class LoginViewController: UIViewController, Subscriber, Trackable {
             return
         }
         saveEvent("login.locked")
+        navigationController?.isNavigationBarHidden = true
```

Comment:
why is this in the lockIfNeeded func

Reply:
it's only needed when walletDisabled is shown